site/pages/create-disruption.tsx

```diff
@@ -1,9 +1,138 @@
-const CreateDisruption = () => {
-    return <></>;
+import React, { ReactElement } from "react";
+import DateSelector from "components/DateSelector";
+import TimeSelector from "components/TimeSelector";
+import { BaseLayout } from "components/layout/Layout"
+import { DisruptionValidity } from "interfaces";
+
+const title = 'Create Disruptions';
+const description = 'Create Disruptions page for the Create Transport Disruptions Service';
+
+interface CreateDisruptionProps {
+    inputs: DisruptionValidity
+}
+
+const CreateDisruption = ({ inputs }: CreateDisruptionProps): ReactElement => {
+    return (
+        <BaseLayout title={title} description={description}>
+            <>
+                <div className="govuk-form-group">
+                    <fieldset className="govuk-fieldset" role="group" aria-describedby="when-is-the-disruption">
+                        <legend className="govuk-fieldset__legend govuk-fieldset__legend--l">
+                            <h1 className="govuk-fieldset__heading">
+                                When is the disruption?
+                            </h1>
+                        </legend>
+                        <legend className="govuk-fieldset__legend govuk-!-padding-top-4">
```

Comment:
Semantically I don't think we should have multiple legends within a fieldset, we should ideally use a structure similar to this example https://design-system.service.gov.uk/patterns/question-pages/#asking-multiple-questions-on-a-page, so "When is the disruption?" would be outside of the fieldsets as a heading and each set of date fields would be in their own fieldsets

Reply:
done